proto/p4/config/p4info.proto

```diff
@@ -21,13 +21,11 @@ import "google/protobuf/any.proto";
 package p4.config;
 
 message P4Info {
-  repeated HeaderField header_fields = 1;
```

Comment:
@teverman I think I covered this in my answer to another comment. There won't be p4info ids for header fields any more if we go through with this update because header fields will not play any part in p4runtime